components/authz-service/storage/v2/ruletype.go

```diff
@@ -0,0 +1,56 @@
+package v2
+
+import (
+	"fmt"
+	"strings"
+)
+
+// RuleType is an enum of the types a project rule can be.
+type RuleType int
+
+const (
+	Node RuleType = iota
+	Event
+)
+
+func (r RuleType) String() string {
+	if r < Node || r > Event {
+		panic(fmt.Sprintf("unknown value from iota RuleType on String() conversion: %d", r))
```

Comment:
Do we have to panic here bc `String()` doesn't return an error?

Reply:
yup